lib/Cargo.toml

```diff
@@ -13,10 +13,18 @@ fuzz = []
 trace = []
 test = []
 
+# Certora feature exposes many of the internal implementation details, so should
+# only be used in formal verification. Ironically, the security testing makes
+# the code less secure.
```

Comment:
make note about conditional compiling even if obvs to any dev

Reply:
I can mention something in a readme. That comment is what a feature is in a Cargo.toml

Reply:
Ya think something for the uninformed on formal verification generally in the readme is useful